api/pom.xml

```diff
@@ -136,6 +136,11 @@
 			<artifactId>servlet-api</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Why do we need to include javax.mail? this module depends on openmrs core, which already bundles mail.